lib/src/user_event/user_event.dart

```diff
@@ -63,6 +63,13 @@ abstract class UserEvent {
   /// [UIEvent.detail](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail)
   /// for more information.
   ///
+  /// ### [skipPointerEventsCheck]
```

Comment:
It looks like this option is also available on `hover`, `unhover`, `selectOptions`, and `deselectOptions` - do we want to add it to those methods in this PR as well?